File: app/src/main/java/com/carlos/grabredenvelope/services/MyService.java
package com.carlos.grabredenvelope.services;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import android.support.annotation.Nullable;

/**
 * Created by 小不点 on 2016/2/23.
 */
public class MyService extends Service{


    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }

    @Override
    public int onStartCommand(Intent intent, int flags, int startId) {
        return super.onStartCommand(intent, flags, startId);
    }


}

File: app/src/main/java/com/carlos/grabredenvelope/services/MyService.kt
package com.carlos.grabredenvelope.services

import android.app.Service
import android.content.Intent
import android.os.IBinder

/**
 * Created by 小不点 on 2016/2/23.
 */
class MyService : Service() {


    override fun onBind(intent: Intent): IBinder? {
        return null
    }

    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
        return super.onStartCommand(intent, flags, startId)
    }


}
